woocommerce/woocommerce
# Fixing incorrect case causing erorrs with case-sensitive environments

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes [#48991](https://github.com/woocommerce/woocommerce/issues/48991) .

> Using create-woo-extension on a case-sensitive system (in my case Linux) results in errors with the PHP autoloader. This issue is intended to capture the correcting the directory and filename casing for compatibility with the autoloader.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Checkout this branch.
2. Take note of the path to `woocommerce/packages/js/create-woo-extension`
3. Run the command npx @wordpress/create-block -t {PATH_TO_CREATE_WOO_EXTENSION} my-extension-name
4. With the `/includes` folder, ensure that the `Admin` directory and `Setup.php` filename are both capitalized.
5. Activate the plugin, there should be no errors.



### Changelog entry



-   [] Automatically create a changelog entry from the details below.


<details>

<summary>Correcting directory and filename casing to address bug with PHP autoloader.</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
